modules/AWS/eks/variables.tf

```diff
@@ -27,12 +27,27 @@ variable "instance_types" {
   type        = list(string)
 }
 
-variable "desired_capacity" {
-  description = "Desired number of nodes that the node group should launch with initially."
+variable "instance_disk_size" {
+  description = "Size of the disk attached to the cluster instance."
```

Comment:
Below we refer to cluster `nodes` and here we refer to them as cluster `instances`. I think its better to stick to one terminology - maybe `nodes`?

Reply:
On second thought maybe `instance` is better, this means you wont have to rename this variable :) and simply update the descriptions for `max_cluster_capacity` and `min_cluster_capacity`